lib/renderer/ipc-renderer-internal.ts

```diff
@@ -20,3 +20,9 @@ ipcRendererInternal.sendTo = function (webContentsId, channel, ...args) {
 ipcRendererInternal.sendToAll = function (webContentsId, channel, ...args) {
   return ipc.sendTo(internal, true, webContentsId, channel, args)
 }
+
+ipcRendererInternal.invoke = async function<T> (channel: string, ...args: any[]) {
+  const { error, result } = await ipc.invoke<T>(internal, channel, args)
```

Comment:
Am I blind, where is `internal` coming from?

Reply:
I think this should just be `true`

Reply:
it's declared at the top of the file, this was done long time ago to "document" the arguments, all other calls are doing the same